composer.json

```diff
@@ -3,7 +3,8 @@
     "description": "Take credit card payments on your store using Stripe.",
     "homepage": "https://woocommerce.com/products/woocommerce-gateway-stripe/",
 		"license": "GPL-2.0",
-		"config": {
+    "type": "wordpress-plugin",
```

Comment:
Please remove the composer package update as it doesn't relate to adding a filter.

Reply:
That has been removed in another PR. #1301